frappe/public/js/frappe/form/quick_entry.js

```diff
@@ -81,7 +81,7 @@ frappe.ui.form.QuickEntryForm = Class.extend({
 
 	validate_for_prompt_autoname: function(){
 		if(this.meta.autoname && this.meta.autoname.toLowerCase()==='prompt') {
-			this.mandatory = [{fieldname:'__name', label:__('{0} Name', [this.meta.name]),
+			this.mandatory = [{fieldname:'__newname', label:__('{0} Name', [this.meta.name]),
```

Comment:
Why is setting to `__newname` required? Looks like the `__name` case is excluded in the below change, so can we leave it to `__name`?